### ISSUE TITLE
Contributor name missing in some demos, leads to empty anchor accName

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#313]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/familysearch/

Avatar has empty alt, but in this case the author field is blank so not accName for the link. Should be David Walp.

**Comment by dstorey** [comment]
Also an issue for https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/coloringbook/

**Comment by dstorey** [comment]
Also https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/ortcdemo/

**Comment by dstorey** [comment]
Also: https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/microphone/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/photocapture/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/compatinspector/

**Comment by dstorey** [comment]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/audiomixer/

**Comment by alrra** [comment]
- [x] https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/familysearch/
- [x] https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/coloringbook/
- [x] https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/ortcdemo/
- [x] https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/microphone/
- [x] https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/photocapture/
- [x] https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/compatinspector/
- [x] https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/audiomixer/
